ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/machine/DevMachineServerProperties.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A., Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Mario Loriedo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api.machine;
+
+import org.eclipse.che.api.core.model.machine.ServerProperties;
+
+/**
+ * Describe development machine server instance.
+ * {@link ServerProperties}
```

Comment:
Curly brackets are used when link is inlined into description. When it is used as a separate tag it has the same syntax as `@author`